src/schema/schema.graphql

```diff
@@ -280,10 +280,36 @@ type AuthResponse {
 	skin: [Skin]
 }
 
+type ScratchCode{
+	ScratchCode: String
+}
+
+type EnableTwoFactorAuthResponse {
+	secret: [Secret]
+	scratchCodes: [ScratchCode]
+	authToken: [TwoFactorAuthToken]
```

Comment:
are we using authtoken anywhere? it will already be available in cookie

Reply:
done